x-pack/legacy/plugins/uptime/server/lib/adapters/monitors/adapter_types.ts

```diff
@@ -5,16 +5,44 @@
  */
 
 import { MonitorChart, MonitorPageTitle } from '../../../../common/graphql/types';
+import { UMElasticsearchQueryFn } from '../framework';
+import { MonitorDetails } from '../../../../common/runtime_types';
+
+export interface GetMonitorChartsDataParams {
+  /** @member monitorId ID value for the selected monitor */
```

Comment:
@justinkambic do these annotations do something? Is there a macro somewhere that does something special with `@member`?

Reply:
It's just a `js-doc` annotation.

![image](https://user-images.githubusercontent.com/18429259/70545711-0428ec80-1b3c-11ea-861d-fc53ada6a6f8.png)

Reply:
https://jsdoc.app/tags-member.html